.vscode/settings.json

```diff
@@ -17,4 +18,8 @@
         "string": "cpp",
         "filesystem": "cpp"
     },
+    "files.associations": {
```

Comment:
What does this do? Is it necessary? (just curious)

Reply:
I think VSCode automatically adds a cpp to file.associations whenever a C++ file with an extension-less or non-standard file extension is opened. When I was looking at the different files in cisTEM, perhaps new entries were generated in files.association because I navigated to that file. Adding just BUILD_OpenMP":true should still work.

Reply:
I see. I don't think we should keep these lines in there - `limits` and `iostream` don't sounds like filenames associated with the cisTEM project. How about just removing that block (at least from this pull request) since it's not related to OpenMP?

Reply:
Okay, I will do that. It is a bit annoying that if someone is developing a new feature using a header file (e.g., limits, vector, iterator, memory, array, random, etc.) or library, settings.json automatically updates in VSCode, which then github tracks. 

I appreciate your feedback!

Reply:
Can you specify what steps exactly led to these being added there? I have not had these things appear in my `settings.json` file... when use vscode to look at say`<limits>`, it seems to know this is a C++ file and doesn't add anything to the `settings.json` file

Reply:
Sure.

1. Open VSCode
2. Open any program, for example refine2D.cpp
3. Right click #include "../../core/core_headers.h" and click Go To Definition
4. Look for #include <limits> and right click and click Go To Definition

Then limits appeared in my settings file. I was confused why it duplicated the file associations section (even VSCode complained) and I had to change that on another branch I am working on for a new feature, but for this openmp branch I changed it to the way you wanted.

Reply:
Thanks - I can reproduce that behavior. Here's what I might suggest: you could prepare a separat PR where you have done this for all the includes, so that the set of associations is complete and won't need to be expanded until a new include is added. Hopefully this will minimize the number of times we have to modify this settings file.